docs/plugins/mapper-annotated-text.asciidoc

```diff
@@ -119,7 +119,7 @@ PUT my_index/_doc/1
 
 PUT my_index/_doc/2
 {
-  "my_field": "[Jeff Beck](Jeff+Beck&Guitarist) plays a strat"<2>
+  "my_field": "[Jeff Beck](Jeff+Beck&Guitarist) plays a start"<2>
```

Comment:
This one refers to [fender stratocaster](https://en.wikipedia.org/wiki/Fender_Stratocaster).